Minio/V4Authenticator.cs

```diff
@@ -202,14 +202,69 @@ private ReadOnlySpan<byte> GenerateSigningKey(string region, DateTime signingDat
     /// <param name="content">Bytes to be hmac computed</param>
     /// <returns>Computed hmac of input content</returns>
     private ReadOnlySpan<byte> SignHmac(ReadOnlySpan<byte> key, ReadOnlySpan<byte> content)
+    {
+        return ComputeHmac256(key.ToArray(), content.ToArray());
```

Comment:
I think this code is the cause of the Functional test failure caught during the build tests.

You need to debug the code and resolve the issue.
When I reverted this change and run the Functional tests with the code suggested in the previous review comments, Functional tests ran successfully.